src/Algebra/Graph/AdjacencyMap.hs

```diff
@@ -348,7 +348,7 @@ vertexSet = Map.keysSet . adjacencyMap
 edgeSet :: Ord a => AdjacencyMap a -> Set (a, a)
 edgeSet = Map.foldrWithKey (\v es -> Set.union (Set.mapMonotonic (v,) es)) Set.empty . adjacencyMap
 
--- | The /postset/ of a vertex is the set of its /direct successors/.
+-- | The /postSet/ of a vertex is the set of its /direct successors/.
```

Comment:
I was actually using 'preset' and 'postset' as terms (not very common ones, but accepted in the Petri Net community), hence no capitalisation.

Inconsistent capitalisation does seem a bit confusing, so if we change to using 'postSet' instead of 'postset' then I suggest that we also drop the slashes around, replacing them with either quotes `'postSet'` or @ sign `@postSet@`.

Reply:
Yep. Made some more commits so that this should be clear now. Feel free to squash-and-merge.